test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -103,6 +103,20 @@ public void dropAllObjects() throws SQLException {
     JdbcFixture.dropAllObjects();
   }
 
+  private String nowPlus(int minutes) {
+    switch (JdbcFixture.DATABASE) {
+      case H2:
+        return "dateadd('minute', " + minutes + ", now())";
```

Comment:
Why did you stick with using now() here instead of keeping the original use of current_timestamp()?